x-pack/plugins/security_solution/public/common/components/events_viewer/event_details_flyout.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlyout, EuiFlyoutBody, EuiFlyoutHeader } from '@elastic/eui';
+import React, { useCallback } from 'react';
+import styled from 'styled-components';
+import deepEqual from 'fast-deep-equal';
+import { useDispatch } from 'react-redux';
+
+import { TimelineExpandedEventType } from '../../../../common/types/timeline';
+import { ColumnHeaderOptions } from '../../../timelines/store/timeline/model';
+import { timelineActions } from '../../../timelines/store/timeline';
+import { BrowserFields, DocValueFields } from '../../containers/source';
+import {
+  ExpandableEvent,
+  ExpandableEventTitle,
+} from '../../../timelines/components/timeline/expandable_event';
+import { useDeepEqualSelector } from '../../hooks/use_selector';
+
+const StyledEuiFlyout = styled(EuiFlyout)`
+  z-index: 9999;
```

Comment:
I think that EUI give us a definition of z-index, I am wondering if it will be better to use that later on.